app/javascript/components/widget-transition-controls.jsx

```diff
@@ -41,8 +41,8 @@ const reducer = (state, action) => {
       };
     case 'startTimer':
       return {
-        msRemainingSinceLastStart: action.msDurationSeconds,
-        startTimestamp: selectIsPaused(state) ? null : now,
+        msRemainingSinceLastStart: action.msDurationRemaining,
```

Comment:
I'm confused by this commit - what does it mean to change `msDurationSeconds` to `msDurationRemaining`, and why are we getting rid of `selectIsPaused(state)` for determining the `startTimestamp`? and by not dispatching the `startTimer` action within `useEffect` of this component, aren't we making a functionality change and not a refactor?

Reply:
The variable name 'milliseconds duration seconds' must just have been a typo in a previous refactor, and I thought msDurationRemaining was more appropriate anyway. For 'selectIsPaused(state) ? null : now', Craig and I decided it's a fair assumption that a widget transition will never occur during a pause, and have to start the timer. I decided on labeling the commit a refactor since the startTimer dispatch wasn't removed from this PR - but instead moved to the final feature commit which creates and passes startTimer.

Reply:
Ok, I think this commit would benefit from a more descriptive commit message. Regarding the `selectIsPaused(state)` change, can't we transition widgets using the arrow keys, even when paused? So widget transitions can occur while paused? Would this result in a timer starting on the next widget while paused if transitioning via arrow keys?

Reply:
Yes, the timer does start on the next widget, but it is not paused. I've tested transitioning widgets from a paused widget, and couldn't find any other possible errors caused by this action. When helios is paused, and then manually transitioned, the next widget loads just as fine before calling startTimer, and then proceeds unpaused. I guess it depends if we want to consider 'pausing, transitioning, and helios automatically unpausing itself' a bug or a feature?

Reply:
I do consider this to be convenient though.

Reply:
This commit is then changing the behavior (if we're paused, we continue to stay paused until resuming, even if transitioning widgets), so it should be in a different commit if that is your intention in changing that functionality. I personally think we should keep the current functionality and stay paused until we resume.